velox/exec/HashAggregation.cpp

```diff
@@ -238,12 +245,43 @@ void HashAggregation::resetPartialOutputIfNeed() {
 }
 
 void HashAggregation::maybeIncreasePartialAggregationMemoryUsage(
-    double aggregationPct) {
+    // If
```

Comment:
@oerling This comment is formatted as one word per line. This is probably accidental. Would you double check?